### PR TITLE
FOV from catalog in target search popup.

### PR DESCRIPTION
**Pull request by rpiaggio**
Automatically set FOV of Aladin in the target search popup if the angular size is present in the catalog.

For the moment this only applies to Aladin within the target search popup, future PR will address initializing the instance of Aladin in the target editor.

### CODE REVIEW

File: explore/src/main/scala/explore/config/SpectroscopyModesTable.scala [hunk 1]
```diff
@@ -387,7 +387,7 @@ object SpectroscopyModesTable extends ItcColumn {
         )
       )((props, _, itcResults, _, _, _, _, _) => { case (wv, sn, rows, range) =>
         implicit val ctx = props.ctx
-        range.value.map(r => updateITCOnScroll(wv, sn, r, rows, itcResults.withEffect)).orEmpty
+        range.value.map(r => updateITCOnScroll(wv, sn, r, rows, itcResults)).orEmpty
```
Comment (cquiroz) [inline explore/src/main/scala/explore/config/SpectroscopyModesTable.scala:390]: ???

**Review by cquiroz — APPROVED** [review]
Great improvement, thanks

File: explore/src/main/scala/explore/targeteditor/AladinContainer.scala [hunk 2]
```diff
@@ -29,7 +29,7 @@ import react.common._
 import scala.annotation.nowarn
 import scala.concurrent.duration._
 
-final case class AladinContainer(
+final case class AladinContainer( // TODO Use target's FOV if not set in user prefs.
```
Comment (toddburnside) [inline explore/src/main/scala/explore/targeteditor/AladinContainer.scala:32]: Depending on the target model we choose, saving the target FOV in user prefs will be more or less difficult. Currently in staging, I don't think it really works with the unshared TargetEnv (although I'd have to look). Maybe we could use target name? With an unshared model, using target id has limitations.

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/config/SpectroscopyModesTable.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/config/SpectroscopyModesTable.scala:390]: Ooops, sorry. I was trying things out and this slipped through.

This is unrelated, but it seems the `withEffect` is not required.

File: explore/src/main/scala/explore/targeteditor/AladinContainer.scala (hunk 2 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/targeteditor/AladinContainer.scala:32]: Understood.

**Pull request merged**